src/command_modules/azure-cli-interactive/azclishell/configuration.py

```diff
@@ -112,7 +112,7 @@ def get_style(self):
 
     def has_feedback(self):
         """ returns whether user has given feedback """
-        return az_config.getboolean('core', 'given feedback')
+        return az_config.has_option('core', 'given feedback') and az_config.getboolean('core', 'given feedback')
```

Comment:
You can do `az_config.getboolean('core', 'given feedback', fallback='false')`.  then don't need to call has_option.